rpm/SPECS/openresty-openssl-asan.spec

```diff
@@ -1,6 +1,6 @@
 Name:               openresty-openssl-asan
-Version:            1.1.0k
-Release:            4%{?dist}
+Version:            1.1.1d
```

Comment:
Because openssl may break API compatibility across major release version numbers, we'd better introduce different package names like `openresty-openssl111` and `openresty-openssl111-asan` to make sure older openresty package versions still work.

Reply:
Very well, I'll make the change.